### DISCUSSION TITLE
¿Puedo hacer otro tipo de páginas?

### DISCUSSION THREAD

**Discussion by PatricioNovellinoLaunchx** [Launch-X-Latam/MisionFrontEnd#374] [Q&A]
Desarrollo la pregunta: yo estoy trabajando en una página que es un proyecto personal de poner una pastelería con mi pareja.
En vez de hacer la práctica de CSS con la consigna pedida, la de js con pokedex y así... ¿Podré centrarme en mi página e ir agregando los diferentes módulos que vamos aprendiendo? Es decir para la entrega de CSS le agrego estilo, para la de js le agrego sus funciones pero todo con la misma página

**Comment by doguedogue** [comment]
Por supuesto, no veo por qué no. Practicarias los conocimientos y estarias llevando a producción tu proyecto, genial 🚀🚀
